[BUG] Error on install with Node 14 LTS

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Not working in node version 14 - using LTS 14.20.1. According to https://github.com/npm/installed-package-contents/releases/tag/v2.0.0 release notes, you are supporting Node version ^14.17.0 

```
❯ nvm install lts/fermium
v14.20.1 is already installed.
Now using node v14.20.1 (npm v6.14.17)
❯ npx npm-check-updates
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /Users/dylankelly/.npm/_npx/70756/lib/node_modules/npm-check-updates/node_modules/@npmcli/installed-package-contents/index.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/dylankelly/.npm/_npx/70756/lib/node_modules/npm-check-updates/node_modules/@npmcli/installed-package-contents/index.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent
```

### Expected Behavior

installs correctly

### Steps To Reproduce

- Use Node version 14 lts/fermium (14.20.1)
- `npm install installed-package-contents`

### Environment

- npm: 6.14.17
- Node: v14.20.1
- OS: Mac OSX 12.6
- platform: Macbook Pro M1 ARM

--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   "author": "GitHub Inc.",
   "main": "lib/index.js",
   "bin": {
-    "installed-package-contents": "index.js"
+    "installed-package-contents": "lib/index.js"
   },
   "license": "ISC",
   "scripts": {

